src/main/java/sirius/biz/tycho/kb/KnowledgeBase.java

```diff
@@ -125,7 +129,15 @@ public final String getFallbackLang() {
      * @return the default language as two-letter ISO code
      */
     public String getFallbackLanguage() {
-        return fallbackLanguage;
+        return !fallbackLanguages.isEmpty() ? null : fallbackLanguages.get(0);
+    }
```

Comment:
Check if we can kill this method and the getFallbackLang above

Reply:
we can kill it since they are not used among scireum-core and ox =)